plugin-flex-ts-template-v2/src/feature-library/supervisor-barge-coach/flex-hooks/components/TeamsView.tsx

```diff
@@ -0,0 +1,24 @@
+import * as Flex from '@twilio/flex-ui';
+import AgentAssistanceTeamsIcon from "../../custom-components/AgentAssistanceTeamsIcon"
+import { isAgentAssistanceEnabled } from '../../config';
+import { FlexComponent } from "../../../../types/feature-loader";
+
+export const componentName = FlexComponent.CallCanvas;
+export const componentHook = function addAgentAssistanceTeamsIcon(flex: typeof Flex, manager: Flex.Manager) {
+
+  if(!isAgentAssistanceEnabled()) return;
+    flex.WorkersDataTable.Content.add(
+    <Flex.ColumnDefinition 
+      key="agent-hand-custom" 
+      header={""} 
```

Comment:
This is where you can fix the Teams view columns getting pushed over. Try adding `style={{width:'22px'}}` (you might want to experiment with different sizes, but that one does seem to look reasonable)